src/core/keyboard_interrupt.c

```diff
@@ -1,29 +1,86 @@
 #define PY_SSIZE_T_CLEAN
 #include "Python.h"
 
+#include "error_handling.h"
 #include "keyboard_interrupt.h"
 #include <emscripten.h>
 
 static int callback_clock = 50;
+_Py_IDENTIFIER(get_event_loop);
+_Py_IDENTIFIER(set_interrupt);
 
+/**
+ * Calls WebLoop.handle_interrupt.
+ * WebLoop.handle_interrupt raises KeyboardInterrupt into all Tasks that are
+ * marked as interruptable.
```

Comment:
What's the practical effect of this? Shouldn't the KeyboardInterrupt exception be raised only from the active task since only one exception can be reported to the user anyway?